compiler/control/OMROptions.cpp

```diff
@@ -539,7 +539,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"disableTLE",                         "O\tdisable transactional lock elision", SET_OPTION_BIT(TR_DisableTLE), "F"},
    {"disableTlhPrefetch",                 "O\tdisable software prefetch on allocation", SET_OPTION_BIT(TR_DisableTLHPrefetch), "F"},
    {"disableTM",                          "O\tdisable transactional memory support", SET_OPTION_BIT(TR_DisableTM), "F"},
-   {"disableTOCForConsts",                "O\tdisable use of the TOC for constants and floats materialization", SET_OPTION_BIT(TR_DisableTOCForConsts), "F"},
+   {"disableTOC",                         "O\tdisable use of the Table of Contents (TOC) on relevant architectures", SET_OPTION_BIT(TR_DisableTOC), "F"},
```

Comment:
Table of Constants is actually the correct one.

Reply:
Oh. So is `Table of Constants` here different than the `Table of Contents` they are talking about here?
https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/assembler/idalangref_und_prg_toc.html

Quoting the first line:
```
The Table of Contents (TOC) of an XCOFF file is analogous to the table of contents of a book.
```

Reply:
Same basic idea, except that the "native" TOC holds a lot more than just constants, while ours was really just for constants. The native TOC is created by the compiler and linker and exists for each module (shared library and executable) in the program. Jitted code doesn't have such a TOC, the closest TOC we have is the TOC for the JIT library, but we can't use that to store our data so we maintain our own. Ours is referred to as a pseudo-TOC in some parts of the code/docs/commit history because it's similar to but not quite the same as the "real" TOC.

Reply:
Ah okay. Thanks for the explanation!

I've reverted back to "Constants" here: a3342a3